packages/services/src/nbconvert/index.ts

```diff
@@ -121,24 +65,24 @@ export namespace NbconvertManager {
   }
 
   /**
-   * The build status response from the server.
+   * A namespace for nbconvert API interfaces.
    */
   export interface IExportFormats {
     /**
      * The list of supported export formats.
      */
     // TODO: should this stay a string, or a typed object
     // that includes an 'output_mimetype' string?
-    readonly exportList: string;
+    [key: string]: { output_mimetype: string };
```

Comment:
going to leave this as output_mimetype, in the event that with @starcruiseromega's changes we can add another 'label' type piece of metadata to this response

Reply:
Sounds good to me.